Dockerfile

```
@@ -0,0 +1,13 @@
+FROM rust:1.65 as build
+RUN apt update -y
+RUN apt install build-essential libhyperscan-dev pkg-config git -y
+WORKDIR "/noseyparker"
+COPY . .
+RUN cargo build --release
+
+FROM debian:11-slim
+COPY --from=build /noseyparker/target/release/noseyparker /usr/bin/noseyparker
+RUN apt update -y
+RUN apt install libhyperscan-dev -y
```

You'll want to combine these two lines into a single `RUN` command for this to behave as expected with Docker caching. See https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

```suggestion
RUN apt update && apt install -y \
    libhyperscan-dev
```